### ISSUE TITLE
Dissconnected nodes! when everything is connected

### ISSUE THREAD

**Issue by Qwerty135246** [sharpie7/circuitjs1#868]
I keeb on getting the 'Dissconnected Nodes!' alert when trying to make a supcircuit with this circuit: https://www.falstad.com/circuit/circuitjs.html?ctz=CQAgjCAMB0l3BWEAWATLBBOAzM5kxVJsAObbEANkhCQRoQFMBaMMAKDARJTEpCI88PQVHYBjWoQGQeCaanxQoseARDMYJSphIB2SmWR7MyNhUjsATlNQC9dhAft2a2eNdsh3yWs5-KYGqWAO5+-KgO4d6QvqHgqDxgmHaEPAHxaTG+ZvwZ7GG54M5FopYAht7Y-Gx2RbUgmOBIrEhBwco07R5hAZh51Y388eQ1iVU1ziODYCV8xcMFvJP89dKWfBBZfF3OO2I2qIrg1NknNAxwS315sedikgH7AWUqaiSYn1-fP52eL7IzqgEMNwD0gSCztgEK5rndoY5-DCxGEnHlkWizvFMQiqIDcZYAJJ49LIwyk1wqOjsYnkgQkHjUEQMzrQaloOxM+mM-HIi6wCDoalhOm4umJHiWIh6BJJFIgEhdeW+ADy7GlsvuivuvkJ7AA9icqHEUJBPm94AZUDhTAhsIQkDA1JgDCDgcpUnl2EA

**Comment by pfalstad** [comment]
yes this seems to be a bug.  the problem is the VCO.  you can work around it like this: https://tinyurl.com/2hxml6jt

**Comment by pfalstad** [comment]
This works now
